app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-copy-tracker.js

```diff
@@ -0,0 +1,48 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.analyticsGa4 = window.GOVUK.analyticsGa4 || {}
+window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analyticsModules || {};
+
+(function (analyticsModules) {
+  'use strict'
+
+  var Ga4CopyTracker = {
+    init: function () {
+      var getSelectedText = this.getSelectedText.bind(this)
```

Comment:
Haven't tested it, but could you instead bind `this` to the eventListener itself, and then you wouldn't need this variable?

E.g. 

```JS
window.addEventListener('copy', function () {
      var text = this.getSelectedText()
      }.bind(this))
```

Reply:
Good shout, implemented.